src/main/java/org/gusdb/sitesearch/service/Service.java

```diff
@@ -63,6 +61,77 @@ public Response getStreamingResults(String body) {
     return handleStreamRequest(getSolr(), new SearchRequest(new JSONObject(body), false, true, false));
   }
 
+  /**
+   * SOLR Response:
+   *
+   * <pre>
+   * {
+   *   "suggest": {
+   *     "default": {
+   *       "<term>": {
+   *         "numFound": 10,
+   *         "suggestions": [
+   *           {
+   *             "term": "<match>",
+   *             "weight": 0,
+   *             "payload": ""
+   *           }
+   *         ]
+   *       }
+   *     }
+   *   }
+   * }
+   * </pre>
+   */
+  @GET
+  @Path("/suggest")
+  @Produces(MediaType.APPLICATION_JSON)
+  public Response getSuggestions(@QueryParam("searchText") String searchText) {
+    // Return an empty list for requests that contain no search text or a search
+    // term that is fewer than 3 characters.
+    if (searchText == null || searchText.isBlank() || searchText.trim().length() < 3)
+      return Response.ok("[]", MediaType.APPLICATION_JSON_TYPE).build();
```

Comment:
Do you need this second arg if you have the @Produces annotation above?

Reply:
Probably not.